.github/workflows/ci.yml

```diff
@@ -48,8 +48,8 @@ jobs:
       fail-fast: false
       matrix:
         java: ['adopt@1.8', 'adopt@1.11']
-        scala: ['2.12.15', '2.13.8', '3.1.0']
-        platform: ['JVM', 'Native']
+        scala: ['2.12.15', '2.13.8', '3.1.1']
```

Comment:
I'm not sure we want to go to 3.1.1 - should be in sync with ZIO and it was 3.1.0 for RC2

Reply:
I see. Let me fix this.

Reply:
Fixed.